examples/include_pkg/build.envd

```diff
@@ -0,0 +1,6 @@
+envdlib = include("https://github.com/kemingy/envdlib")
```

Comment:
Should we embed the envdlib into the envd binary? git clone may take a long time, especially in China.

Reply:
It will be cached in `~/.cache/envd/envdlib/<repo>`.

I tried to reduce the repo size with `depth=1` but it failed to `git pull`, see https://github.com/go-git/go-git/issues/305.

BTW, `envdlib` should be very small since it only contains README and envd files.

Reply:
If users want to use tensorboard, do they need to add `envdlib = include("https://github.com/kemingy/envdlib")` into the build.envd?

Reply:
I think it is not necessary if the envdlib is maintained by us. The feature can be used for third-party libs.

Reply:
Make sense. So shall we maintain the `envdlib` in `envd` or use git submodule?
I can create another PR to evaluate the `envdlib` and insert into `envd`.

Reply:
git submodule LGTM.

/cc @VoVAllen 

And maybe we can maintain an example about using third-party plugins.

Should we merge the PR first?

Reply:
Can merge this one if it LGTY